extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -373,29 +382,30 @@ public static function doRetroactiveAutoblock( Block $block, array &$blockIds )
 	/**
 	 * Register tables that need to be updated when a user is renamed
 	 *
-	 * @param DatabaseBase $dbw
-	 * @param int $userId
-	 * @param string $oldUsername
-	 * @param string $newUsername
+	 * @param DatabaseBase      $dbw
+	 * @param int               $userId
+	 * @param string            $oldUsername
+	 * @param string            $newUsername
 	 * @param UserRenameProcess $process
-	 * @param int $wgCityId
-	 * @param array $tasks
+	 * @param int               $wgCityId
+	 * @param array             $tasks
+	 *
 	 * @return bool
 	 */
 	public static function onUserRenameLocal( $dbw, $userId, $oldUsername, $newUsername, $process, $wgCityId, array &$tasks ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$dbw, $userId, $oldUsername, $newUsername, $process, $wgCityId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)